scripts/create_sql_lite_dump.py

```diff
@@ -0,0 +1,125 @@
+import click
+import click_config_file
+import isb_lib.core
+from isamples_metadata import Transformer
+from isb_lib.models.isb_core_record import ISBCoreRecord
+from isb_web.isb_solr_query import ISBCoreSolrRecordIterator
+import requests
+from sqlmodel import SQLModel, create_engine, Session
+import logging
+
+# Use this to map between the solr column names and the sqlite column names and control what is
+# written to the dump.
```

Comment:
Here we declare which fields we want to include in the dump, as well as rename them when appropriate.